src/views/ModelPlan/ReadOnly/ParticipantsAndProviders/index.tsx

```diff
@@ -100,14 +100,14 @@ const ReadOnlyParticipantsAndProviders = ({ modelID }: { modelID: string }) => {
     >
       <div className="display-flex flex-justify flex-align-start">
         <h2 className="margin-top-0 margin-bottom-4">{t('heading')}</h2>
-        {status && <TaskListStatusTag status={status} />}
+        <TaskListStatusTag status={status} />
       </div>
 
       <div className="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light">
         <ReadOnlySection
           heading={t('whoAreParticipantsQuestion')}
           list
-          listItems={participants?.map(translateParticipantsType)}
+          listItems={participants.map(translateParticipantsType)}
```

Comment:
Another example of code clean up.

because data is for sure not going to be undefined (as per line 42 in this file), participants will never be undefined.